broker/cmd/docker-compose/plus-2c-15kmsgs.yml

```diff
@@ -0,0 +1,36 @@
+# on top of docker-compose.yml adds 2 more consumers and increases the producer rate to 15k msgs/s
+version: "3"
+
+services:
+  producer1:
+    environment:
+      CONSTANT_TICK_COUNT: 15000
+      BATCH_SIZE: 2500
+
+  consumer1:
```

Comment:
This it has the only environment? it does not have an image, and other properties, how does it works?